src/components/Button/GoogleButton.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { ReactComponent as GoogleIcon } from '@/assets/icons/googleIcon.svg';
+
+const GoogleLoginButton: React.FC = () => {
+  const GoogleButtonClick = () => {
+    const redirectUrl = process.env.OAUTH_Google_REDIRECT_URI;
+    if (redirectUrl) {
+      window.location.href = redirectUrl;
+    } else {
+      console.error('찾을 수 없음');
+    }
+  };
+
+  return (
+    <GoogleBtn onClick={GoogleButtonClick}>
+      <IconContainer>
+        <GoogleIcon />
+      </IconContainer>
+      <ButtonText color="#242424">Google로 계속하기</ButtonText>
+      <div style={{ width: 24, height: 24 }} />
+    </GoogleBtn>
+  );
+};
+
+const GoogleBtn = styled.button`
+  width: 100%;
+  height: 56px;
+  padding: 0 24px;
+  display: inline-flex;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: 8px;
+  cursor: pointer;
+  border: 1px solid #dfdfdf;
+  background: white;
+  &:hover {
+    background: white;
+    filter: brightness(80%);
+  }
+`;
+
+const IconContainer = styled.div`
+  width: 24px;
+  height: 24px;
+  padding: 1px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const ButtonText = styled.p`
+  margin-left: 16px;
+  color: ${(props) => props.color};
```

Comment:
style 컴포넌트에 props를 넘겨줄 때 이름앞에 $를 붙이고 타입도 지정하는 방식이 관례인 것 같더라구요! 혹시 이렇게 변경하는 것에 대해서는 어떻게 생각하시나요?
```suggestion
const ButtonText = styled.p<{$color: string}>`
  margin-left: 16px;
  color: ${(props) => props.$color};
```

아래에는 styled components 공식 문서에 있는 예시입니다!
```
// Create an Input component that'll render an <input> tag with some styles
const Input = styled.input<{ $inputColor?: string; }>`
  padding: 0.5em;
  margin: 0.5em;
  color: ${props => props.$inputColor || "#BF4F74"};
  background: papayawhip;
  border: none;
  border-radius: 3px;
`;

// Render a styled text input with the standard input color, and one with a custom input color
render(
  <div>
    <Input defaultValue="@probablyup" type="text" />
    <Input defaultValue="@geelen" type="text" $inputColor="rebeccapurple" />
  </div>
);
```

Reply:
아하 넵! 이렇게 다 수정하겠습니다!